packages/owl-bot-post-processor/src/helpers.ts

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021 Google LLC
```

Comment:
`helpers.ts` is too generic a name for this file.

Reply:
I've renamed it to `core.ts`, the idea is that these are the core functions that will be shared by both the command line application and the web application.

I'm finding the approach of going command line first like this has helped me debug _a ton_, before I make any attempt to run it as a service.

Reply:
+1 one to having a commandline